Python/compile.c

```diff
@@ -1116,9 +1143,14 @@ stack_effect(int opcode, int oparg, int jump)
             /* 0 in the normal flow.
              * Restore the stack position and push 6 values before jumping to
              * the handler if an exception be raised. */
-            return jump ? 6 : 0;
+            return jump ? 3 : 0;
+        case SETUP_CLEANUP:
+            return jump ? 4 : 0;
```

Comment:
I'm guessing 3 of those are the same as SETUP_FINALLY.  What's the fourth?

Reply:
`lasti`, the instruction offset where the exception was raised.